src/applications/vaos/new-appointment/components/CommunityCarePreferencesPage.jsx

```diff
@@ -15,6 +14,160 @@ import {
 } from '../redux/actions';
 import { getFormPageInfo } from '../../utils/selectors';
 import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import { createSelector } from 'reselect';
```

Comment:
I think I'd prefer to keep this address field code that was moved into here in its own file. Generally we keep these schema/uiSchema definitions separate from components, when they're this complicated. It could still be in the new-appointment feature folder, though.